utils/utils.go

```diff
@@ -75,3 +75,7 @@ func ConvertJson2Map(js []byte) (map[string]string, error) {
 	err := json.Unmarshal(js, &result)
 	return result, err
 }
+
+func S(s string) *string {
```

Comment:
why are we using string pointers instead of string? I did not see any place where we are comparing to nil. Remove it.